### ISSUE TITLE
Can't find where to edit screen dimensins

### ISSUE THREAD

**Issue by chirani** [lightning-js/solid-starter-template#1]
The default screen size I assume 1920x1080.
Is there a way to change it or make it more dynamic?

**Comment by chiefcll** [comment]
Yes - you're right about the default. We recommend keeping it that size and then changing 
```
Config.rendererOptions = {
deviceLogicalPixelRatio: 0.6666667,
}
```
That is for 720p and `deviceLogicalPixelRatio: 2` is good for 4k. 

Yes - If you'd like to open a PR to detect screen size and do this automatically that would be great... 👍
